core/models.py

```diff
@@ -47,7 +47,9 @@ class Meta:
         verbose_name_plural = 'Organizers'
 
     def __str__(self):
-        return '{0} ({1})'.format(self.get_full_name(), self.email)
+        if self.first_name == '' and self.last_name == '': 
+            return '{0}'.format(self.email)
+        return '{0}'.format(self.get_full_name())
```

Comment:
I think that the e-mail should still be attached to the full_name if it is there. I am affraid it can be used somewhere for copy&paste reasons for e-mails or something :)

Sorry if I sound picky, I hope you will forgive me :)